xsd2json.js

```diff
@@ -524,6 +524,12 @@ function processChoice(obj, parent, key) {
     }
 }
 
+function removeUnique(obj, parent, key) {
+    if (obj[xsPrefix + "unique"] != null) {
```

Comment:
Thanks for the PR! Are we sure this test is correct? 

```suggestion
    if (typeof obj[xsPrefix + "unique"] !== 'undefined') {
```

Reply:
Thanks for the fast reply! The test is correct to the best of my knowledge, does something seem off to you?